listenbrainz/webserver/templates/index/playlists-management.html

```diff
@@ -0,0 +1,34 @@
+{%- extends 'base.html' -%}
+{%- block title -%}Playlists management - ListenBrainz{%- endblock -%}
+{%- block content -%}
+  <h2 class="page-title">Playlists managementg </h2>
+  {# Playlists management information #}
+  <h3>Synchronizing Playlists</h3>
+  <p>
+    There are many ways to sync your own playlists to ListenBrainz:
+  </p>
+  <h4>Music players</h4>
+  <ul>
+    <li><em><a href="https://www.foobar2000.org/">Foobar2000</a></em>, full-fledged audio player for Windows: <a href="https://github.com/regorxxx/Playlist-Manager-SMP"><code>Playlist-Manager-SMP</code></a></li>
+  </ul>
+
+  <h3>Synchronizing via third-party music services</h3>
+  <p>
+    <a href={{ url_for('profile.music_services_details') }}>Connect your ListenBrainz account</a> and import/export your playlists to some third-party services.
```

Comment:
We are likely to soon add an option  to import playlists from Spotify, but in the meantime this is a good start